app/cmd/restore_to_file.go

```diff
@@ -13,7 +13,9 @@ import (
 	"github.com/urfave/cli"
 	"gopkg.in/cheggaaa/pb.v2"
 
-	iutil "github.com/longhorn/go-iscsi-helper/util"
+	lhexec "github.com/longhorn/go-common-libs/exec"
+	lhtypes "github.com/longhorn/go-common-libs/types"
+	commonUtils "github.com/longhorn/go-common-libs/utils"
```

Comment:
nit: suggest using the same prefix for the package alias. for example, here we can just use `commonExec`, `commonTypes` and `commonUtils`.

This minor consistent alignment can be improved later.

Reply:
Fixed.